memory_742.x

```diff
@@ -0,0 +1,107 @@
+MEMORY
+{
+  /* This file is intended for parts in the STM32H742/742v families. (RM0433) */
+  /* - FLASH and RAM are mandatory memory sections.                           */
+  /* - The sum of all non-FLASH sections must add to 692K total device RAM.   */
```

Comment:
The only family that supports re-assigning memory around the map is STM32H735, so I don't think there's any value in stating this sum for the other families